tests/evaluation/test_elo_evaluator.py

```diff
@@ -0,0 +1,193 @@
+    return Evaluator(
+        in_memory_dataset_repository,
+        in_memory_run_repository,
+        in_memory_evaluation_repository,
+        "Testing",
+        dummy_eval_logic,
+    )
+
+
+@fixture
+def dummy_qa_input() -> SingleChunkQaInput:
+    return SingleChunkQaInput(chunk=TextChunk(""), question="", language=Language("en"))
+
+
+@fixture
+def dummy_qa_output() -> SingleChunkQaOutput:
+    return SingleChunkQaOutput(answer=None, highlights=[])
+
+
+@fixture
+def qa_outputs() -> Sequence[SingleChunkQaOutput]:
+    return [
+        SingleChunkQaOutput(answer=answer, highlights=[])
+        for answer in [
+            "Surface micromachining builds microstructures.",
+            "Surface micromachining builds microstructures. This is done by deposition and etching structural layers over a substrate.",
+            "Surface micromachining builds microstructures by deposition and etching structural layers over a substrate. This is different from Bulk micromachining, in which a silicon substrate wafer is selectively etched to produce structures.",
+        ]
+    ]
+
+
+@fixture
+def qa_setup(
+    in_memory_dataset_repository: InMemoryDatasetRepository,
+    in_memory_run_repository: InMemoryRunRepository,
+    qa_outputs: Sequence[SingleChunkQaOutput],
+) -> Tuple[Sequence[str], str]:
+    qa_input_text = TextChunk(
+        """Surface micromachining builds microstructures by deposition and etching structural layers over a substrate.[1] This is different from Bulk micromachining, in which a silicon substrate wafer is selectively etched to produce structures."""
+    )
+    #
+    qa_input = SingleChunkQaInput(
+        chunk=qa_input_text, question="What is micromachining?", language=Language("en")
+    )
+    expected_output = "Surface micromachining builds microstructures by deposition and etching structural layers over a substrate."
+    #
+    example_id = "some-example-id"
+    dataset_id = in_memory_dataset_repository.create_dataset(
+        examples=[
+            Example(input=qa_input, expected_output=expected_output, id=example_id)
+        ],
+        dataset_name="some-example-dataset-name",
+    ).id
+    #
+    run_ids = [f"some-run-id-{i}" for i in range(len(qa_outputs))]
+    for i, output in enumerate(qa_outputs):
+        in_memory_run_repository.store_example_output(
+            example_output=ExampleOutput(
+                run_id=run_ids[i],
+                example_id=example_id,
+                output=output,
+            )
+        )
+        in_memory_run_repository.store_run_overview(
+            RunOverview(
+                dataset_id=dataset_id,
+                id=run_ids[i],
+                start=utc_now(),
+                end=utc_now(),
+                failed_example_count=0,
+                successful_example_count=len(qa_outputs),
+                description="runner",
+            )
+        )
+    return run_ids, dataset_id
```

Comment:
this method is only used once and you don't use the dataset id

Reply:
Good catch, removed `dataset_id`from the return